python/ray/includes/gcs_client.pxi

```diff
@@ -78,7 +78,7 @@ cdef class NewGcsClient:
     @property
     def cluster_id(self) -> ray.ClusterID:
         cdef CClusterID cluster_id = self.inner.get().GetClusterId()
-        return ray.ClusterID.from_binary(cluster_id.Binary())
+        return ray.ClusterID(cluster_id.Binary())
```

Comment:
Anything inherits from UniqueID has `from_binary()` so we don't need to change here?

Reply:
yes just to keep consistency. do you want me to revert this?

Reply:
Yea, let's just add the missing `from_binary()` instead of changing to using constructor. This can prevent the bug from happening next time.